aten/src/ATen/native/sparse/cuda/SparseCUDABlas.cu

```diff
@@ -81,7 +76,7 @@ cusparseOperation_t convertTransToCusparseOperation(char trans) {
   }
 }
 
-#if !defined(_MSC_VER) && defined(__CUDACC__) && CUSPARSE_VERSION >= 10301 // CUDA release >= 10.2 and not windows
+#if defined(__CUDACC__) && CUSPARSE_VERSION >= 11000
```

Comment:
Hmm, your change disables this functionality on Linux with CUDA-10.2, which is not ideal, is it?
```suggestion
#if defined(__CUDACC__) && (CUSPARSE_VERSION >= 11000 || (!defined(_MSC_VER) && CUSPARSE_VERSION >= 10301))
```

Reply:
You can check cusparse details at https://github.com/pytorch/pytorch/pull/42412. On linux cuda 10.2, using new or old API should have no big differences. They are both functional. This is just to make the macro straightforward.